.github/workflows/ci.yml

```diff
@@ -22,9 +24,33 @@ jobs:
           node-version: '14.x'
       - run: npm ci
       - run: npx lockfile-lint --path package-lock.json --allowed-schemes "https:" "git+https:" --allowed-hosts npm github.com
-      - run: npm run test
-        env:
-          ADYEN_INTEGRATION_CONFIG: ${{ secrets.ADYEN_INTEGRATION_CONFIG }}
+      - run: npm run check
+      - run: npm run cover:unit
+      - name: prepare extensions folder
+        run: |
+          echo $ADYEN_INTEGRATION_CONFIG >> .extensionrc
+          mkdir -p sources && cp index.googleFunction.js index.js
+          cp -R {index.js,.extensionrc,src,resources,package.json} sources
```

Comment:
I checked README in deployment examples for google functions and `extensionrc` is not mentioned there, though it is used in `zip-google-function` script. Can you update the README?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/890/commits/6fec71b11d35148d60adf4b957b087242c342c8a